test/common/stats/utility_fuzz_test.cc

```diff
@@ -8,9 +14,68 @@ namespace Envoy {
 namespace Fuzz {
 
 DEFINE_FUZZER(const uint8_t* buf, size_t len) {
-  const absl::string_view string_buffer(reinterpret_cast<const char*>(buf), len);
-  Stats::Utility::sanitizeStatsName(string_buffer);
+  {
+    const absl::string_view string_buffer(reinterpret_cast<const char*>(buf), len);
+    Stats::Utility::sanitizeStatsName(string_buffer);
+  }
+  if (len < 4) {
```

Comment:
can you put some ASSERT checks that you don't run out of bytes throughout? I don't have a high degree of confidence that this check ensures you have enough bytes for all your Consumes below.

Reply:
Do you mean the ASSERT checks are needed every place where we are going to consume the string?

Reply:
@jmarantz

Reply:
Just drive-by if there are no bytes left for Consumes, ConsumeBool returns default false and ConsumeRandomLengthString returns an empty string. 

I think createScope can take an empty string, but I haven't looked into where it' used in pool.add

Reply:
Yup, the idea though is that it's not a very interesting fuzzer if it just fuzzing a bunch of empty strings, so I think we should tune the sizings so that we get non-empty strings generally.

Reply:
I further divide the remaining data on line 58 to reduce the chance to get empty string. Do you think that will solve the problem?

Reply:
Actually I looked this over again and it's fine; you don't need to divide that one by 3.